### ISSUE TITLE
Consider adding a new reporter or a flag to make viewing changes easier

### ISSUE THREAD

**Issue by p-himik** [liquidz/antq#240]
With the table reporter and changes enabled, the changes URLs are listed below the table. It makes it inconvenient to go back and forth between the table and the changes, always searching for the right line.

Perhaps a new reporter or a flag could be added to make the changes URLs appear beside the corresponding entries in the report table, right on the same row.

**Comment by liquidz** [comment]
@p-himik Thanks for your suggestion.

Originally, it was planned to display it on the same row.
But when actually displayed, the table became hard to read due to the length of the URLs, so I've changed it to the current form.

Therefore, it is easy to display it on the same row depending on the option.
I made a [feature/changes-in-table](https://github.com/liquidz/antq/tree/feature/changes-in-table) branch and added `--changes-in-table` option for trial.
Could you try this branch?

**Comment by liquidz** [comment]
@p-himik Ah, as a tool, newlines are handled correctly, so it seems that the string received by tools.cli has already been escaped. 
```
clojure -Tantq outdated :error-format '"File: {{file}}\nLatest: {{latest-version}}"'
```

I've fixed in the same branch. Could you try?

**Comment by liquidz** [comment]
@p-himik Thanks for your confirmation!
I'll prepare for release.

`logback-classic`'s pom.xml does not have `SCM` section.
* https://repo1.maven.org/maven2/ch/qos/logback/logback-classic/1.4.14/logback-classic-1.4.14.pom
* https://maven.apache.org/pom.html#SCM

antq refers the SCM section to detect the source repository URL, so antq cannot show the changes URL for `logback-classic` for now.

**Comment by liquidz** [comment]
@p-himik Sorry, I've been unwell.

> Not sure whether there's a better way to handle latest-name to avoid those extra ->, but they don't really bother me.

Now `dev` branch uses [Mustache](https://mustache.github.io) as a template engine for trial, and we can specify error-format as follows.

```
"File:    {{file}}\nName:    {{name}}{{#latest-name}} -> {{.}}{{/latest-name}}\nVersion: {{version}}{{#latest-version}} -> {{.}}{{/latest-version}}\nChanges: {{changes-url}}\n"
```

**Comment by p-himik** [comment]
Oh, cool, thanks!

Hope you feel better.

**Comment by liquidz** [comment]
@p-himik Just released v2.8.1165 :)
